elastica/boundary_conditions.py

```diff
@@ -5,6 +5,7 @@
     "FreeRod",  # Deprecated: remove v0.3.0
     "OneEndFixedBC",
     "OneEndFixedRod",  # Deprecated: remove v0.3.0
+    "ConfigurableConstraint",
```

Comment:
```suggestion
    "GeneralConstraint",
```

Reply:
Implemented in https://github.com/GazzolaLab/PyElastica/pull/143/commits/4c06b0e850b5d5fb647394771572c35a9f0c641e